aspen/configuration/__init__.py

```diff
@@ -41,6 +41,7 @@
     , 'project_root':       (None,                  parse.identity)
     , 'renderer_default':   ('stdlib_percent',      parse.renderer)
     , 'show_tracebacks':    (False,                 parse.yes_no)
+    , 'colorize':           (True,                  parse.yes_no)
```

Comment:
How about `colorize_tracebacks` to be a) more specific and b) parallel with `show_tracebacks`?